src/cleancall.c

```diff
@@ -35,14 +43,23 @@ static void call_exits(void* data) {
 
     void (*fn)(void*) = (void (*)(void*)) R_ExternalPtrAddrFn(CAR(cb));
     void *data = (void*) EXTPTR_PTR(CDR(cb));
+    int nonlocal = LOGICAL(EXTPTR_TAG(CDR(cb)))[0];
 
     // Check for empty pointer in preallocated callbacks
     if (fn) {
+      if (! nonlocal || ! datawpr->success)
```

Comment:
```suggestion
      if (!nonlocal || !(datawpr->success))
```

Spaces after unary operators is non-standard style I think.

Reply:
Maybe `!early_handler || !state->success` would be easier to read?